scripts/deploy/common.rb

```diff
@@ -31,3 +31,52 @@ def open_url(url)
     puts "Opening url #{url}"
     `open #{url}`
 end
+
+def delete_git_branch(branch_name)
+    # Ensure we are not on the same branch that we're trying to delete
```

Comment:
```suggestion
    # Ensure we are on the same branch that we're trying to delete
```

Reply:
I think the original is accurate -- this does something like
```
git checkout master
git branch -D release/version_number
```

if you try to delete `release/version_number` while on that branch, it often fails

Reply:
maybe I should rename the @branch variable to be something more specific? I'm not sure what to call it. This is the branch that you're deploying from -- maybe `deploy_branch`?

Reply:
Yeah, I think I missed the difference between `@branch` and `branch_name`. Makes sense now.

Reply:
renamed to deploy_branch so it's more obvious that there are two different branches being used in this method